From 21b1996a90bd32e7c8d80c75c8e547bb9e6eeab9 Mon Sep 17 00:00:00 2001
From: Alexander Mikhaylenko <alexm@gnome.org>
Date: Tue, 21 Feb 2023 20:39:14 +0400
Subject: [PATCH] graph-view: Don't use gtk_widget_get_color()

That func was added in 4.9.x and since GTK doesn't ship its own vapi we'd
have to bundle Vala as well. Instead, just use the deprecated one.
---
 src/graph-view.vala | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/graph-view.vala b/src/graph-view.vala
index fbd209f..585e412 100644
--- a/src/graph-view.vala
+++ b/src/graph-view.vala
@@ -171,7 +171,7 @@ public class Elastic.GraphView : Adw.Bin {
         Cairo.Context cr,
         double alpha_multiplier
     ) {
-        Gdk.RGBA rgba = darea.get_color ();
+        Gdk.RGBA rgba = darea.get_style_context ().get_color ();
 
         cr.set_source_rgba (
             rgba.red, rgba.green, rgba.blue,
-- 
2.39.1

